test/mason/masonInit/masonInitPathTest.chpl

```diff
@@ -0,0 +1,13 @@
+use FileSystem;
```

Comment:
Could you add some more test cases, such as the ones I tried in the review? You could follow what I did like so:

```chpl
const newArgs = ['new', 'testSrc'];
masonNew(args)
runCommand(' rm -rf testSrc/src');
const initArgs = ['init', 'testSrc'];
masonInit(initArgs);
// check to make sure `src/` and `src/testSrc.chpl` were created
runCommand('rm -rf testSrc');
```